From d795e92bd12705d0ce506c4ac900b0ccb7bdc69b Mon Sep 17 00:00:00 2001
From: Nicolas Fella <nicolas.fella@gmx.de>
Date: Tue, 18 Apr 2023 23:54:02 +0200
Subject: [PATCH] Persist imported VPN connections on disk

Otherwise they disappear after a reboot

BUG: 468666


(cherry picked from commit 024035190982ac5957310ef78fbc34b9c51766c2)
---
 libs/handler.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libs/handler.cpp b/libs/handler.cpp
index 625cbabd7..57fb9f2e3 100644
--- a/libs/handler.cpp
+++ b/libs/handler.cpp
@@ -378,7 +378,7 @@ void Handler::addConnection(NMConnection *connection)
 
     nm_client_add_connection2(client,
                               nm_connection_to_dbus(connection, NM_CONNECTION_SERIALIZE_ALL),
-                              NM_SETTINGS_ADD_CONNECTION2_FLAG_IN_MEMORY,
+                              NM_SETTINGS_ADD_CONNECTION2_FLAG_TO_DISK,
                               nullptr,
                               true,
                               nullptr,
-- 
GitLab

